### PR TITLE
test(sanic): run tests for multiple versions

### PR DESCRIPTION
**Pull request by majorgreys**
## Description

Though tox included sanic versions, those versions were not in fact being tested. There is a bit of work needed to get the integration to support 18.12 (LTS) and 19.3 so those have been removed here.

## Checklist
- [x] Entry added to `CHANGELOG.md`.
- [x] Tests provided; and/or
- [ ] Description of manual testing performed and explanation is included in the code and/or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

**Comment by majorgreys** [comment]
For both 18.12 and 19.3, the Sanic tests are failing with this error:

```
ERROR    sanic.root:handlers.py:119 Exception occurred while handling uri: unknown
NoneType: None
ERROR    asyncio:plugin.py:76 Exception in callback <bound method HttpProtocol.response_timeout_callback of <sanic.server.HttpProtocol object at 0x109300ca0>>
handle: <TimerHandle cancelled>
Traceback (most recent call last):
  File "uvloop/cbhandles.pyx", line 267, in uvloop.loop.TimerHandle._run
  File "/Users/tahir.butt/go/src/github.com/DataDog/dd-trace-py/.tox/sanic_contrib-py38-sanic1812/lib/python3.8/site-packages/sanic/server.py", line 213, in response_timeout_callback
    self.write_error(ServiceUnavailable("Response Timeout"))
  File "/Users/tahir.butt/go/src/github.com/DataDog/dd-trace-py/.tox/sanic_contrib-py38-sanic1812/lib/python3.8/site-packages/sanic/server.py", line 488, in write_error
    self.log_response(response)
  File "/Users/tahir.butt/go/src/github.com/DataDog/dd-trace-py/.tox/sanic_contrib-py38-sanic1812/lib/python3.8/site-packages/sanic/server.py", line 357, in log_response
    self.request.method, self.request.url
  File "/Users/tahir.butt/go/src/github.com/DataDog/dd-trace-py/.tox/sanic_contrib-py38-sanic1812/lib/python3.8/site-packages/sanic/request.py", line 301, in url
    (self.scheme, self.host, self.path, None, self.query_string, None)
  File "/Users/tahir.butt/go/src/github.com/DataDog/dd-trace-py/.tox/sanic_contrib-py38-sanic1812/lib/python3.8/site-packages/sanic/request.py", line 260, in scheme
    self.app.websocket_enabled
AttributeError: 'NoneType' object has no attribute 'websocket_enabled'
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Changelog
 
+- test(sanic): run tests for multiple versions
```
Comment (Kyle-Verhoog) [inline CHANGELOG.md:3]: I don't think this changelog entry is necessary.

**Review by Kyle-Verhoog — COMMENTED** [review]
:+1: let's be sure to track the work for adding the support for the older versions

**Comment by Kyle-Verhoog** [comment]
@majorgreys seems like the sanic tests are all failing 🤔 https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/2861/workflows/70ba03cd-ffa9-4ae4-bcb2-770410ab2f66/jobs/347810

**Review by Kyle-Verhoog — APPROVED** [review]

**Comment by Kyle-Verhoog** [comment]
@majorgreys we should also update the corp site docs

**Pull request merged**